lib/Runtime/Library/JavascriptRegularExpression.cpp

```diff
@@ -1516,6 +1561,8 @@ using namespace Js;
         case PropertyIds::source:
         case PropertyIds::options:
             IS_WRITABLE(!scriptConfig->IsES6RegExPrototypePropertiesEnabled())
+        case PropertyIds::dotAll:
+            IS_WRITABLE(!scriptConfig->IsES9RegExDotAllEnabled())
```

Comment:
Why does this one not follow the pattern of the others? In every other method the corresponding case looks like `(scriptConfig->IsES9RegExDotAllEnabled() && !scriptConfig->IsES6RegExPrototypePropertiesEnabled())`.